src/scripts/main.js

```diff
@@ -1,3 +1,285 @@
 'use strict';
 
-// write code here
+// New Employee Section
+
+// Create a form
+const form = document.createElement('form');
+
+form.classList.add('new-employee-form');
+document.body.append(form);
+
+// Create input fields and labels
+const nameInput = document.createElement('input');
+const nameLabel = document.createElement('label');
+const positionInput = document.createElement('input');
+const positionLabel = document.createElement('label');
+const officeSelect = document.createElement('select');
+const selectLabel = document.createElement('label');
+const ageInput = document.createElement('input');
+const ageLabel = document.createElement('label');
+const salaryInput = document.createElement('input');
+const salaryLabel = document.createElement('label');
+const button = document.createElement('button');
+
+const tableHeaders = [nameInput, positionInput, ageInput, salaryInput];
+
+const cities
+= [`Tokyo`, `Singapore`, `London`, `New York`, `Edinburgh`, `San Francisco`];
+
+// Add attributes to all inputs
+for (const header of tableHeaders) {
+  header.name = 'name';
+  header.type = 'text';
+}
+
+ageInput.type = 'number';
+salaryInput.type = 'number';
+button.type = 'submit';
+
+// Create a collection of options with properties for the select tag
+for (let i = 0; i < cities.length; i++) {
+  const option = document.createElement('option');
+
+  option.value = cities[i];
+  option.text = cities[i];
+  officeSelect.add(option, officeSelect[i]);
+}
+
+// Set data attributes
+nameInput.setAttribute('data-qa', 'name');
+positionInput.setAttribute('data-qa', 'position');
+officeSelect.setAttribute('data-qa', 'office');
+ageInput.setAttribute('data-qa', 'age');
+salaryInput.setAttribute('data-qa', 'salary');
+
+// Text values for labels
+const textName = 'Name: ';
+const textPosition = 'Position: ';
+const textOffice = 'Office: ';
+const textAge = 'Age: ';
+const textSalary = 'Salary: ';
+const buttonText = 'Save to table';
+
+// Add some text value to a label
+nameLabel.textContent = textName;
+positionLabel.textContent = textPosition;
+selectLabel.textContent = textOffice;
+ageLabel.textContent = textAge;
+salaryLabel.textContent = textSalary;
+button.textContent = buttonText;
+
+// Add inputs to the coresponding labels
+nameLabel.append(nameInput);
+positionLabel.append(positionInput);
+selectLabel.append(officeSelect);
+ageLabel.append(ageInput);
+salaryLabel.append(salaryInput);
+
+// Appention of the labels to the form
+form.append(nameLabel);
+form.append(positionLabel);
+form.append(selectLabel);
+form.append(ageLabel);
+form.append(salaryLabel);
+form.append(button);
```

Comment:
another way to do this (in my humble opinion more declarative and neat) is to use insertAdjacentHTML method and add markup with all the necessary information embedded inside, but the you wrote is also valid
![image](https://user-images.githubusercontent.com/57598825/151168680-41ecd017-4f95-4ae1-91c6-3cb519ae3b30.png)